### PR TITLE
Add Avenue.app v4.2.1

### PR DESCRIPTION
**Pull request by xburelle**
Add Resolume Avenue, VJ Software

### CODE REVIEW

File: Casks/resolume-avenue.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask :v1 => 'resolume-avenue' do
+  version '4.2.1'
+  sha256 '996306b9ff0b4ec99e8b5338e8153c07e5ff194ae3ec7f350d791b8b336deb95'
+
+  url 'http://download.resolume.com.s3.amazonaws.com/Resolume_Avenue_4_2_1_Installer.dmg'
+  name 'Resolume'
+  name 'Resolume Avenue'
+  homepage 'https://resolume.com'
+  license :freemium
```
Comment (vitorgalvao) [inline Casks/resolume-avenue.rb]: Is this a free app that works indefinitely but with limitations that can be removed by paying?

**Comment by alebcay** [comment]
> Because you receive a serial number by email it is impossible to return this so your payment is not refundable. Download Resolume and try it as long as you like. The only restriction (until you have entered your serial number) is the Resolume logo watermarked on the video output and a robot voice in the audio.

**Comment by vitorgalvao** [comment]
Thank you, @alebcay.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 227a38bb72b44fdc5b02bedc19a5adc94f05b2dc. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**